lib/active_merchant/billing/gateways/element.rb

```diff
@@ -194,7 +194,7 @@ def add_transaction(xml, money, options = {})
       def add_terminal(xml, options)
         xml.terminal do
           xml.TerminalID '01'
-          xml.CardPresentCode 'UseDefault'
+          xml.CardPresentCode options[:card_present_code] if options[:card_present_code]
```

Comment:
Since the remote tests still pass, I assume that Element doesn't care if we're not passing this tag at all?

Reply:
That's a really good point. I like the idea of updating with a fallback case. The only argument for **not** using "UseDefault" as a fallback is that there seems to be a discrepancy for what the default value should be. There is some recent documentation for the Element gateway that suggests "Default" is the default value. (see below)

![Screen Shot 2020-05-12 at 8 26 04 PM](https://user-images.githubusercontent.com/13825990/81758132-f848aa00-948e-11ea-9d5c-b7f26e260eaa.png)

I'm going to try to find a ground truth as to what exactly the default value should be (and if it changes), and update with that value as a fallback. (I imagine it's _UseDefault_, since that's what seems to work at the moment)

 While I'm at it, going to update the commit message according to [this](https://gist.github.com/jknipp/dbe9896a7fd74a0138218ae875243512)

Reply:
Well, it's definitely UseDefault

Reply:
Made "UseDefault" fallback case: https://github.com/activemerchant/active_merchant/blob/8a5cdb9329e2ec7d38acdd23e83c5d680a4db2f5/lib/active_merchant/billing/gateways/element.rb#L197

I moved the entry in CHANGELOG for this issue back into HEAD and updated the Pull Request to track this one. Is moving it into HEAD okay here?

Also updated the commit message to conform to styling.

Reply:
Changelog entries should always be located under the `HEAD` section. After you rebase this branch, you'll need to resolve conflicts within the CHANGELOG file and ensure that your entry is still located at the bottom of the `HEAD` section.